akka-docs/src/main/paradox/common/binary-compatibility-rules.md

```diff
@@ -93,9 +93,10 @@ If you accidentally mix Akka versions, for example through transitive
 dependencies, you might get a warning at run time such as:
 
 ```
-You are using version 2.6.6 of Akka, but it appears you (perhaps indirectly) also depend on older versions
-of related artifacts. You can solve this by adding an explicit dependency on version 2.6.6 of the
-[akka-persistence-query] artifacts to your project. 
+You are using version 2.6.6 of Akka, but it appears you (perhaps indirectly) also depend on older versions 
+of related artifacts. You can solve this by adding an explicit dependency on version 2.6.6 of the 
+[akka-persistence-query] artifacts to your project. Here's a complete collection of detected 
+artifacts: (2.5.3, [akka-persistence-query]), (2.6.6, [akka-actor, akka-cluster]).
```

Comment:
real example would include more stuff here, but good to keep it simple in the docs